proto/osmosis/superfluid/tx.proto

```diff
@@ -67,4 +69,13 @@ message MsgLockAndSuperfluidDelegate {
   ];
   string val_addr = 3;
 }
-message MsgLockAndSuperfluidDelegateResponse { uint64 ID = 1; }
\ No newline at end of file
+message MsgLockAndSuperfluidDelegateResponse { uint64 ID = 1; }
+
+// ===================== MsgUnPoolWhitelistedPool
+message MsgUnPoolWhitelistedPool {
+  string sender = 1 [ (gogoproto.moretags) = "yaml:\"sender\"" ];
+  uint64 pool_id = 2 [ (gogoproto.moretags) = "yaml:\"pool_id\"" ];
+  uint64 lock_id = 3 [ (gogoproto.moretags) = "yaml:\"lock_id\"" ];
+}
```

Comment:
So I think we should remove `lock_id`, and apply this for all locks for this sender and LP share denom pair.

Reply:
Agreed with this decision 👍